lib/systems/include/TorqueControllers.h

```diff
@@ -181,4 +189,38 @@ class TorqueControllerLoadCellVectoring : public TorqueController<TC_LOAD_CELL_V
         const AnalogConversion_s &rrLoadCellData);
 };
 
+class TorqueControllerSimpleLaunch : public TorqueController<TC_SIMPLE_LAUNCH>
+{
+private:
+    const float launch_ready_accel_threshold = .1;
+    const float launch_ready_brake_threshold = .2;
+    const float launch_ready_speed_threshold = 5.0; // m/s
+    const float launch_go_accel_threshold = .9;
+    const float launch_stop_accel_threshold = .5;
+
+    uint16_t max_torque_target = 2142;
```

Comment:
https://github.com/hytech-racing/MCU/blob/f34e01403756fa4b082c7da7de2caa7f49d87701/lib/systems/include/TorqueControllers.h#L19

Update the value in `TorqueControllers.h` and use it instead for consistency

Reply:
These values differ from what was used on HT07 which were just int numbers. Is the float number scaled up when it is put into the CAN message?

Reply:
Nevermind, saw the answer in later comment